WordPress/src/main/java/org/wordpress/android/ui/accounts/SignInFragment.java

```diff
@@ -1244,10 +1258,16 @@ public void certificateTrusted() {
                 showGenericErrorDialog(getResources().getString(R.string.xmlrpc_missing_method_error));
                 break;
             case XMLRPC_BLOCKED:
-                // TODO: show a better error message
+                //TODO: use this to help the user a bit:  pass the Helpshift page ID or section ID
+                //on the rest of the error cases in this switch
```

Comment:
Can you do the extra step and add other FAQ id/section to other errors when applicable?